TestIcebergDistributedHive.testLargeQuery times out

For failures like this I wonder if we should mark the test skipped rather than failed?

2024-03-15T17:37:40.6187470Z [ERROR] Tests run: 2075, Failures: 1, Errors: 0, Skipped: 23, Time elapsed: 1,670.442 s <<< FAILURE! - in TestSuite
2024-03-15T17:37:40.6189307Z [ERROR] com.facebook.presto.iceberg.hive.TestIcebergDistributedHive.testLargeQuery  Time elapsed: 0.037 s  <<< FAILURE!
2024-03-15T17:37:40.6191864Z org.testng.internal.thread.ThreadTimeoutException: Method com.facebook.presto.tests.AbstractTestQueries.testLargeQuery() didn't finish within the time-out 30000
2024-03-15T17:37:40.6194239Z 	at org.testng.internal.invokers.MethodInvocationHelper.invokeWithTimeoutWithNoExecutor(MethodInvocationHelper.java:336)
2024-03-15T17:37:40.6196204Z 	at org.testng.internal.invokers.MethodInvocationHelper.invokeWithTimeout(MethodInvocationHelper.java:294)
2024-03-15T17:37:40.6197726Z 	at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:679)
2024-03-15T17:37:40.6199024Z 	at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:220)
2024-03-15T17:37:40.6200371Z 	at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
2024-03-15T17:37:40.6201760Z 	at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:945)
2024-03-15T17:37:40.6203179Z 	at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:193)
2024-03-15T17:37:40.6204312Z 	at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
2024-03-15T17:37:40.6205124Z 	at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
2024-03-15T17:37:40.6205875Z 	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
2024-03-15T17:37:40.6206860Z 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
2024-03-15T17:37:40.6207426Z 	at java.lang.Thread.run(Thread.java:750)
2024-03-15T17:37:40.6207674Z 
2024-03-15T17:37:41.0939866Z [INFO]

--- a/presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestQueries.java
+++ b/presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestQueries.java
@@ -5666,7 +5666,7 @@ public abstract class AbstractTestQueries
         assertQueryFails(stringBuilder.toString(), "Query results in large bytecode exceeding the limits imposed by JVM|Compiler failed");
     }
 
-    @Test(timeOut = 30_000)
+    @Test(timeOut = 60_000)
     public void testLargeQuery()
     {
         StringBuilder query = new StringBuilder("SELECT * FROM (VALUES ROW(0, '0')");

